### ISSUE TITLE
Change CAS EDL delegation to use JWT instead of Oauth Tokens

### ISSUE THREAD

**Issue by bsatoriu** [MAAP-Project/ZenHub#480]

**Comment by gchang** [comment]
The CAS in the dit env is pointing to ops urs. Change that to the uat urs to test out. I think you have access to the maap application in uat urs, let me know if you don't. Also, I should respond to the urs team about when we think we can get this resolved. After taking a look at it, please let me know and I can pass it on. Thx.

**Comment by bsatoriu** [comment]
@gchang I'm setting up a local instance of [maap-auth-cas](https://github.com/MAAP-Project/maap-auth-cas) to test the JWT method. I can have this done by end of next week at latest. It will likely be earlier, but want to give ourselves plenty of  time to test and deploy to GCC.

**Comment by bsatoriu** [comment]
I added the CAS jwks file and generated a new keystore file to upgrade the tokens from Oauth to JWT. I tested both on DIT and UAT (the latter was tested by temporarily switching from ops to uat EDL and unchecking the  box labeled `Require temporary waiver to keep OAuth token length at 64 chars`.

@gchang when ready, simply uncheck this box from the Ops app, and then I can run a quick test by simply logging into NASA URS from MAAP.
![Screen Shot 2021-11-18 at 12 37 48 PM](https://user-images.githubusercontent.com/27687558/142492833-e0db5b8d-f243-4260-88b0-1008ce944e8a.png)
